apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentTypeSection.tsx

```diff
@@ -186,7 +187,11 @@ const AssignContentTypeSection = (props: Props) => {
           )}
         </>
       ) : (
-        <Spinner variant="primary" />
+        <Flex marginTop="none" fullWidth={true}>
+          <Skeleton.Container svgHeight={147}>
```

Comment:
Yeah I do agree, I don't like using magic numbers! Though in this case with the loading containers, this is meant to reflect the exact height of the component that eventually renders, so I don't foresee another solution besides this, since SVG height needs to be pretty explicit? Though I will think on it! 

Luckily the container that eventually renders here is 150px so I can really just remove this here, but this pattern can certainly be seen with the other loading states that have been implemented previously on the page.